sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -80,16 +80,19 @@ export default abstract class HttpBaseClient {
     throw error;
   }
 
-  // TODO: ONCE ALL URL'S ARE UPDATED, THEN THIS
-  // FUNCTION CAN BE A PRIVATE FUNCTION AND CALLED
-  // BY getCallTyped OR PATCH OR POST FUNCTION
-  // WHEN NEW MODULES ARE ADDED, UPDATE THE CASE ALSO
-  // SO, THAT THIS FUNCTION CAN BE USED TO ADD HOOD TO ALL
-  //  API URL
+  // TODO: Once all url's are updated, then this
+  // function can be a private function and called
+  // by getCallTyped or patch or post function
   public addClientRoot(url: string) {
     switch (AuthService.shared.authClientType) {
       case ClientIdType.AEST:
-        return `aest/${url}`;
+        return `${ClientTypeBaseRoute.AEST}/${url}`;

```

Comment:
Thanks for doing this. After the story, you can add this information to wiki for other team members to follow. Ignore if you have already done 😊

Reply:
added pls have a look

Reply:
https://github.com/bcgov/SIMS/wiki/Development-Standards-and-Decisions#adding-client-root